portfolio/src/main/java/com/google/sps/servlets/ContactMeListsServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+import com.google.sps.data.ContactMe;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for listing tasks. */
+@WebServlet("/list-contactMe")
+public class ContactMeListsServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    Query<Entity> query =
+        Query.newEntityQueryBuilder().setKind("contactMe").setOrderBy(OrderBy.desc("timestamp")).build();
+    QueryResults<Entity> results = datastore.run(query);
+
+    List<ContactMe> contactMes = new ArrayList<>();
```

Comment:
is there a reason you're instantiating the ArrayList using the List interface instead of writing "ArrayList<ContactMe> contactMes = new ArrayList<>();" ? Unless there's a reason for it, I would change this to ArrayList<ContactMe> contactMes = new ArrayList<>();
here's an explanation of ArrayList vs List if you're unfamiliar - https://www.geeksforgeeks.org/difference-between-list-and-arraylist-in-java/